docs/data_management/tutorial/globus_group_tutorial.md

```diff
@@ -0,0 +1,399 @@
+
+## Tutorial Prerequisites
+
+For these tutorials, you will need your BlazerID or [XIAS ID](../../account_management/xias/index.md) and password to authenticate using UAB Single Sign-On (SSO).
+
+## How Do I Get onto the Globus Web App?
+
+To learn how to get onto the Globus Web App [Globus Tutorials for Individual Researchers Page](./globus_individual_tutorial.md#how-do-i-get-onto-the-globus-web-app). Please visit that link and then return here when you have finished. Once you are familiar with
+
+[Return to the top](#globus-tutorials-for-research-groups).
+
+## How Do I Install Globus Connect Personal?
+
+Globus Connect Personal (GCP) is available to install on the following Operating Systems. Linux is also available, but we do not cover the installation process here. If you need to install GCP on Linux, please refer to the [official documentation](https://docs.globus.org/globus-connect-personal/install/linux/).
+
+Managing data for a Research Core? Please stop here. You should know that Globus Connect Personal is _not_ suitable for production use with customers. Please [Contact Support](../../help/support.md#how-do-i-create-a-support-ticket) to discuss setting up Globus Connect Server for your Research Core.
+
+Representing a lab? Globus Connect Personal may be suitable for your use case. If you transfer data infrequently, or transfer small amounts of data, then GCP is probably sufficient. If you frequently transfer large amounts of data, then Globus Connect Server can enable higher data transfer rates through parallelization. Please [Contact Support](../../help/support.md) if you want to discuss installation of Globus Connect Server.
+
+GCP installation instructions for:
+
+- [Windows](#installing-gcp-on-windows)
+- [MacOS](#installing-gcp-on-macos)
+
+[Return to the top](#globus-tutorials-for-research-groups).
+
+### Installing GCP On Windows
+
+The following steps will guide you to install Globus Connect Personal (GCP) on your computer running Windows.
+
+1. Navigate to the [Globus Connect Personal official page](https://www.globus.org/globus-connect-personal) and scroll down to find the GCP Windows version. Click on the "INSTALL NOW" button in the red box in the image below to be taken to the official installation instructions and download link for GCP for Windows.
```

Comment:
It would be better to highlight the 'INSTALL NOW' button on the image with a red box or remove the text "red box" from the main text. For consistency with the macOS installation, marking the button with a red box would be preferable.

Reply:
Done